webclient/app/src/app/commons/appTemplateDialog/AppTemplateDialog.jsx

```diff
@@ -139,24 +139,24 @@ function AppTemplateDialog(props) {
                 autoComplete="off"
             >
                 <ErrorAlert alert={alert} onClose={closeAlert} />
-                <SimpleValidatedTextField
+                <ValidatedTextField
                     fullWidth
                     label={t("appTemplateDialog.location") + "*" }
                     value={internalAppTemplate.location}
                     name="location" 
                     onChange={handleChange}
-                    iscreate={+isCreateDialog}
-                    errortext={t("appTemplateDialog.location.error")}
+                    isCreate={isCreateDialog}
+                    helperText={t("appTemplateDialog.location.error")}
                     regex={RegexConfig.appTemplateLocation}
                 />
-                <SimpleValidatedTextField 
+                <ValidatedTextField 
                     fullWidth 
                     label={t("appTemplateDialog.branch")} 
                     value={internalAppTemplate.branch} 
                     name="branch" 
                     onChange={handleChange}
-                    iscreate={+isCreateDialog}
-                    errortext={t("appTemplateDialog.branch.error")}
+                    isCreate={isCreateDialog}
+                    helperText={t("appTemplateDialog.branch.error")}
                     regex={RegexConfig.appTemplateBranch}
```

Comment:
Why not add an ``isMandatory`` tag to the props? This could simplify the process

Reply:
we can add isMandatory later on - anyway the string length has to be checked.

Reply:
finally fixed javascript isCreate attribute error in addition